homeassistant/components/volvooncall.py

```diff
@@ -123,6 +126,13 @@ def __init__(self, hass, vin, attribute):
         self._state.entities[self._vin].append(self)
 
     @property
+    def entity_id(self):
```

Comment:
Thank you for feedback - do I understand correctly that everything is fine if it is kept as it is, i.e. `entity_id` is specified in class `VolvoEntity`, which is defined in `components/volvooncall`?